src/prefect/server/api/flow_runs.py

```diff
@@ -455,3 +456,82 @@ async def set_flow_run_state(
         response.status_code = status.HTTP_200_OK
 
     return orchestration_result
+
+
+@router.post("/{id}/input", status_code=status.HTTP_201_CREATED)
+async def create_flow_run_input(
+    flow_run_id: UUID = Path(..., description="The flow run id", alias="id"),
+    key: str = Body(..., description="The input key"),
+    value: str = Body(..., description="The value of the input"),
```

Comment:
I love the simplicity, sometimes I forget that FastAPI will produce schema classes for us based on the signature of the API like this